index.html

```diff
@@ -1035,11 +1019,11 @@ <h4>Exchange Examples</h4>
     autonumber
     Note right of H: Start exchange
     W->>I: Initiate
-    Note right of W: POST /exchanges/123 - HTTP request to start exchange (e.g., send credentials, get credentials)
+    Note right of W: POST /workflows/123/exchanges/123 - HTTP request to start exchange (e.g., send credentials, get credentials)
     I->>W: Verifiable Presentation Request (VPR)
     Note left of I: VPR includes method of interaction, for purposes of exchange
     W->>I: Verifiable Presentation (VP)
-    Note right of W: POST /exchanges/123/session-abc - sent via interaction mechanism to meet requirements of exchange
+    Note right of W: POST /workflows/123/exchanges/123/session-abc - sent via interaction mechanism to meet requirements of exchange
```

Comment:
This URL with `session-abc` isn't a thing anymore, but wouldn't hold up the PR for it:

```suggestion
    Note right of W: POST /workflows/123/exchanges/123/session-abc - sent via interaction mechanism to meet requirements of exchange
```

Reply:
If it's retained...
```suggestion
    Note right of W: POST /workflows/123/exchanges/123/session-abc &mdash; sent via interaction mechanism to meet requirements of exchange
```